src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataService.java

```diff
@@ -41,6 +44,8 @@
     private final ResponseMapper responseMapper;
     private final SettlementMapper settlementMapper;
     private final UserService userService;
+    private final JsonMapper jsonMapper;
+    private final ReferenceNumberService referenceNumberService;
 
     @Autowired
     public CoreCaseDataService(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS00107)